oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,213 @@
+CCs abide by their Responsibilities. For more on the CC roles, and Rights & Responsibilities, see:
+
+- `Core Contributor Role Definitions <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2759460357/Core+Contributor+Role+Definitions>`_
+- `Core Contributor Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2952003698/Core+Contributor+Rights+Responsibilities+excluding+code+contributors>`_
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
+
+Note that roles vary in what they need from members; the wiki will be the most up to date source
+of what each role entails and what types of commitments are expected.
+
+.. _program administration:
+
+Core Contributor Program Administration
+---------------------------------------
+
+The Core Contributor Program is administered by The Center for Re-imagining Learning (tCRIL). One or more
+persons at tCRIL (the "Program administrator(s)") work with the community to make sure the program runs
+smoothly: this entails facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and abide by the Code of Conduct.
+Program administrators are empowered to monitor Core Contributor conduct, particularly whether they follow
+the "3 Cs" (Commitment, Conduct, Caliber), and may issue reminders, suspensions, and even removal from
+the Program. Program administrators can be contacted at ``cc-program-admins@tcril.org``.
+
+The tCRIL administrator(s) are the point people for making sure that needed agreements are signed - this includes
+a Program Agreement between the CC and tCRIL; for code contributors, an additional CLA; and for
+new entities joining the community, an Entity Contributor Agreement. We are additionally asking that all employers
+of CCs sign the `Revised Declaration of Commitment to the Core Contributor Program <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3216900524>`_
+to demonstrate that they are willing to engage with and commit resources to the Open edX project. We
+encourage a commitment of 20 hours/month from each of an organization's Core Contributors, however, we
+recognize that circumstances differ and a different commitment may be more appropriate. Before signing
+the agreement, please reach out to the Program Administrators to discuss.
+
+Questions about the Program can be directed to the tCRIL administrator(s) via the ``#core-contributors`` Slack room
+in the `Open edX Slack <https://openedx.slack.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+New Core Contributors are determined via a nomination process - CCs may nominate
+new members, or existing community members may put themselves forward for a role.
+
+Current project CCs should always be on the lookout for contributors who demonstrate the
```

Comment:
What does "project" mean in this context?

Reply:
Yeah that phrasing makes no sense. What about simply `Current CCs`?